lametro/templates/lametro/index.html

```diff
@@ -136,5 +136,29 @@ <h2>
             collapseActivity();
             return false;
         });
+
+        $('#search-bar').autocomplete({
+          serviceUrl: '{% url "autocomplete" %}',
+          paramName: 'query',
+          transformResult: function(response) {
+            var response = JSON.parse(response);
+            return {
+              suggestions: $.map(response.termHints, function(d) {
+                return {'value': d.name, 'data': d.id};
+              })
+            }
+          },
+          onSelect: function(suggestion) {
+            $.when(
+              $.get('/topic/', { 'guid': suggestion.data }))
+                .then(function(response) {
+                  if (response.status_code == 200) {
```

Comment:
this is such an improvement over the jquery ui autocomplete! great find. can you add this autocomplete functionality to the search bar on the search page, too?